src/randoop/main/GenTests.java

```diff
@@ -777,6 +786,7 @@ private void addClassLiterals(ComponentManager compMgr, List<Class<?>> allClasse
 
   /**
    * Execute the init routine (if user specified one)
+   * @param phase
```

Comment:
Empty Javadoc for this silences the compiler warning but doesn't help a programmer...